woocommerce/woocommerce
# Marketplace styling tweaks

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

- Changed illustrations for extensions and themes no results.
- Added bottom border to theme images in product cards, as many themes have white backgrounds.
- Reduced font size and font weight of suggested products titles in no results.



Closes 18377-gh-Automattic/woocommerce.com.
Partially closes 18420-gh-Automattic/woocommerce.com



### How to test the changes in this Pull Request:

1. Check out this branch and run the build: `pnpm run build --filter=woocommerce`.
2. View an empty search result in the marketplace, for example http://localhost:8888/wp-admin/admin.php?page=wc-admin&term=asfd&tab=search&path=%2Fextensions.
3. Note that the illustrations for extensions and themes are updated.
4. Note that the titles "Most popular extensions" and "Our favorite themes" have font size 16px and font weight 600.
5. View the themes tab http://localhost:8888/wp-admin/admin.php?page=wc-admin&tab=themes&path=%2Fextensions.
6. Note that the theme images in the product cards have a bottom border.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Small styling changes to empty search results and theme product cards.

</details>
